### ISSUE TITLE
Allow for multiple metrics

### ISSUE THREAD

**Issue by blake-varden** [tflearn/tflearn#423]
Right now it appears as if you can only display 1 metric, however, there are many cases where we might want to see multiple metrics.  I.e. look at individual class accuracy.

**Comment by aymericdamien** [comment]
Yes, it would indeed be great to add that! If you are interested in it, please help. Otherwise we can add this feature to our roadmap and release it later!
